fdbcli/ChangeFeedCommand.actor.cpp

```diff
@@ -106,91 +174,29 @@ ACTOR Future<bool> changeFeedCommandActor(Database localDb,
 		}
 
 		wait(updateChangeFeed(localDb, tokens[2], ChangeFeedStatus::CHANGE_FEED_CREATE, range));
+		return true;
 	} else if (tokencmp(tokens[1], "stop")) {
 		if (tokens.size() != 3) {
 			printUsage(tokens[0]);
 			return false;
 		}
 		wait(updateChangeFeed(localDb, tokens[2], ChangeFeedStatus::CHANGE_FEED_STOP));
+		return true;
 	} else if (tokencmp(tokens[1], "destroy")) {
 		if (tokens.size() != 3) {
 			printUsage(tokens[0]);
 			return false;
 		}
 		wait(updateChangeFeed(localDb, tokens[2], ChangeFeedStatus::CHANGE_FEED_DESTROY));
+		return true;
 	} else if (tokencmp(tokens[1], "stream")) {
-		if (tokens.size() < 3 || tokens.size() > 5) {
-			printUsage(tokens[0]);
-			return false;
-		}
-		Version begin = 0;
-		Version end = std::numeric_limits<Version>::max();
-		if (tokens.size() > 3) {
-			int n = 0;
-			if (sscanf(tokens[3].toString().c_str(), "%" PRId64 "%n", &begin, &n) != 1 || n != tokens[3].size()) {
-				printUsage(tokens[0]);
-				return false;
-			}
-		}
-		if (tokens.size() > 4) {
-			int n = 0;
-			if (sscanf(tokens[4].toString().c_str(), "%" PRId64 "%n", &end, &n) != 1 || n != tokens[4].size()) {
-				printUsage(tokens[0]);
-				return false;
-			}
-		}
-		if (warn.isValid()) {
-			warn.cancel();
-		}
-		state Reference<ChangeFeedData> feedData = makeReference<ChangeFeedData>();
-		state Future<Void> feed = localDb->getChangeFeedStream(feedData, tokens[2], begin, end);
-		state Future<Void> versionUpdates = requestVersionUpdate(localDb, feedData);
-		printf("\n");
-		try {
-			state Future<Void> feedInterrupt = LineNoise::onKeyboardInterrupt();
-			loop {
-				choose {
-					when(Standalone<VectorRef<MutationsAndVersionRef>> res =
-					         waitNext(feedData->mutations.getFuture())) {
-						for (auto& it : res) {
-							for (auto& it2 : it.mutations) {
-								fmt::print("{0} {1}\n", it.version, it2.toString());
-							}
-						}
-					}
-					when(wait(feedInterrupt)) {
-						feedInterrupt = Future<Void>();
-						feed.cancel();
-						feedData = makeReference<ChangeFeedData>();
-						break;
-					}
-				}
-			}
-			return true;
-		} catch (Error& e) {
-			if (e.code() == error_code_end_of_stream) {
-				return true;
-			}
-			throw;
-		}
+		return handleStreamCommand(localDb, tokens, warn);
```

Comment:
It might be great if a `const std::unordered_map<std::string_view, decltype(some_general_handle_function_type)>` could help defeat this `if/elseif/else` monster.